app/Jobs/SendSignupToPhoenix.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Rogue\Jobs;
+
+use Rogue\Models\Signup;
+use Rogue\Services\Phoenix;
+use Rogue\Services\Registrar;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class SendSignupToPhoenix extends Job implements ShouldQueue
+{
+    use InteractsWithQueue, SerializesModels;
+
+    /*
+     * Signup instance.
+     *
+     * @var Rogue\Models\Signup
+     */
+    protected $signup;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Signup $signup)
+    {
+        $this->signup = $signup;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle(Phoenix $phoenix, Registrar $registrar)
```

Comment:
Could you type hint `$phoenix` and `Registrar` in the constructor so they are instantiated when the class gets built? Similar to what we do in `SendReportbackToPhoenix`